src/main/java/org/sagebionetworks/web/client/jsinterop/react/HasStyle.java

```diff
@@ -27,25 +27,27 @@ public void setVisible(boolean visible) {
     if (visible) {
       if (this.props != null && this.props.style != null) {
         this.props.style.delete("display");
-        this.props.style =
+        // Clone the style object so React treats it as a new prop and properly re-renders.
+        setStyle(
```

Comment:
Could this not be simplified to just
setStyle(this.props.style)
(since setStyle does the clone)?  I'm probably missing something

Reply:
No you're right, I guess I'm just a bit slow today